app/components/Home/AllAmbassadors.jsx

```diff
@@ -1,37 +1,104 @@
 import React from 'react'
 import { Link } from 'react-router'
 import styled from 'styled-components'
-import { Flex } from '@rebass/grid'
-
-import imgBasile from '../../assets/ambassadors/basile.jpg'
-import imgDorian from '../../assets/ambassadors/dorian.jpg'
+import { Flex, Box } from '@rebass/grid'
+import { withNamespaces } from 'react-i18next'
+import { fontWeight } from 'styled-system'
+import imgBenjamin from '../../assets/ambassadors/benjamin.png'
+import imgFlorence from '../../assets/ambassadors/florence.jpg'
 import imgFrederic from '../../assets/ambassadors/frederic.jpg'
-import imgThimothee from '../../assets/ambassadors/timothee.jpg'
+import imgBasile from '../../assets/ambassadors/basile.jpg'
+import imgMathieu from '../../assets/ambassadors/mathieu.jpg'
 
 const AMBASSADORS = [
-  { name: 'Frédéric Bouffetier', username: 'DocFred', img: imgFrederic },
-  { name: 'Timothée Rolland', username: 'Troplent', img: imgThimothee },
-  { name: 'Dorian Cazottes', username: 'DodoLaSoudure', img: imgDorian },
-  { name: 'Basile Asti', username: 'Basile', img: imgBasile },
+  {
+    firstName: 'Benjamin',
+    lastName: 'Piouffle',
+    username: 'Betree',
+    img: imgBenjamin,
+    title1: 'benjaminPiouffleTitle1',
+    title2: 'benjaminPiouffleTitle2',
+  },
+  {
+    firstName: 'Florence',
+    lastName: 'Chatelot',
+    username: 'Miragide',
+    img: imgFlorence,
+    title1: 'florenceChatelotTitle1',
+    title2: 'florenceChatelotTitle2',
+  },
+  {
+    firstName: 'Frédéric',
+    lastName: 'Bouffetier',
+    username: 'DocFred',
+    img: imgFrederic,
+    title1: 'fredericBouffetierTitle1',
+    title2: 'fredericBouffetierTitle2',
+  },
+  {
+    firstName: 'Basile',
+    lastName: 'Asti',
+    username: 'Basile',
+    img: imgBasile,
+    title1: 'basileAstiTitle1',
+    title2: 'basileAstiTitle2',
+  },
+  {
+    firstName: 'Mathieu',
+    lastName: 'Cerf',
+    username: 'Nevjoia',
+    img: imgMathieu,
+    title1: 'basileAstiTitle1',
+    title2: 'basileAstiTitle2',
```

Comment:
Should not have basile in key name if it's generic. I can suggest:

```suggestion
    title1: 'titleAmbassador',
    title2: 'titleCA',
```